clients/ts/signalr-protocol-msgpack/tests/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
The tests need their own `tsconfig.json` for VSCode IntelliSense and squigglies to work. The `tsconfig.json` in the root of the package directory excludes the tests because we don't want them ending up in the npm package, so VSCode dutifully considers them not part of any project and uses the default settings which can't find `@aspnet/signalr`, etc.